src/ctap/mod.rs

```diff
@@ -1521,25 +1517,26 @@ mod test {
         let user_immediately_present = |_| Ok(());
         let mut ctap_state = CtapState::new(&mut rng, user_immediately_present);
 
-        let mut extension_map = BTreeMap::new();
-        extension_map.insert("hmac-secret".to_string(), cbor_bool!(true));
-        let make_extensions = Some(Extensions::new(extension_map));
+        let make_extensions = Some(MakeCredentialExtensions {
+            hmac_secret: true,
+            cred_protect: None,
+        });
         let mut make_credential_params = create_minimal_make_credential_parameters();
         make_credential_params.extensions = make_extensions;
         assert!(ctap_state
             .process_make_credential(make_credential_params, DUMMY_CHANNEL_ID)
             .is_ok());
 
         let pk = sk.genpk();
-        let hmac_secret_parameters = cbor_map! {
-            1 => cbor::Value::Map(CoseKey::from(pk).0),
-            2 => vec![0; 32],
-            3 => vec![0; 16],
```

Comment:
Not a bug necessarily, because it should be able work with any byte vector, but the current auth is 16 bytes long. The 32 was copied from data_formats.rs and I changed all to a consistent 16 bytes length.